### DISCUSSION TITLE
rose stem seg fault

### DISCUSSION THREAD

**Discussion by Adrian-Lock** [MetOffice/simulation-systems#241] [UM]
I had a single model crash (for group=all) in a branch I'm wanting to lodge (https://code.metoffice.gov.uk/trac/um/wiki/ticket/7705/TicketSummary) which didn't seem to make sense.  So I just tried running the same app for the vn13.6 trunk
>  rose stem --group=spice_gcc_ifort_um_high_omp-n48_ga7_amip_altio
and got the same seg fault error (http://fcm1/cylc-review/view/frlk?&suite=vn13.6_um_trunk&no_fuzzy_time=0&path=log/job/1/atmos-spice_gcc_ifort_um_high_omp-n48_ga7_amip_altio-4x2-noios-2t/01/job.err).  Any ideas what's going on?  Is it me or has something gone wrong on spice?

**Comment by jennyhickson** [comment]
Hi Adrian,

We haven't been seeing that failure around - and notably it passed this weekend in our regular testing. Is this something you're seeing consistently?

Thanks
Jenny

**Reply by Adrian-Lock** [reply]
Oh dear, so that probably means it's me, as I just tested vn13.6 trunk again and it still crashes with a seg fault, see:
http://fcm1/cylc-review/taskjobs/frlk/?suite=vn13.6_um_trunk

**Reply by Adrian-Lock** [reply]
Except I tried just retriggering the task and it ran fine - so "just" an intermittent fault, which is a pain!

**Reply by jennyhickson** [reply]
That is frusturating. If there's a pattern of always failing first time and passing on a retrigger then that might give a clue, but if you're the only one hitting it then its going to be hard to track down.
